### PR TITLE
Fix must gather timeout

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #649

In this PR I am fixing the exception issue with CommandFailed.
Also adding timeout for must-gather execution for 600 seconds.

### CODE REVIEW

File: ocs_ci/ocs/utils.py [hunk 1]
```diff
@@ -650,6 +655,13 @@ def collect_ocs_logs(dir_name):
...
-        occli.exec_oc_cmd(cmd, out_yaml_format=False)
+        occli.exec_oc_cmd(
+            cmd, out_yaml_format=False, timeout=must_gather_timeout
+        )
     except CommandFailed as ex:
         log.error(f"Failed during must gather logs! Error: {ex}")
+    except TimeoutExpired as ex:
+        log.error(
+            f"Timeout {must_gather_timeout}s for must-gather reached, command"
+            f" exited with error: {ex}"
```
Comment (ebenahar) [inline ocs_ci/ocs/utils.py]: I think you need {str(ex)} instead of {ex}

Comment (petr-balogh) [reply ocs_ci/ocs/utils.py]: no need at all Elad, see output from ipython, the python do the conversion itself and calling __str__:
```
In [1]: try:
   ...:     raise Exception("This is my exception for Elad :)")
   ...: except Exception as ex:
   ...:     print(f"Exception raised: {ex}")
   ...:
Exception raised: This is my exception for Elad :)
```

Here you have another examples which are redundant:
```

In [2]: try:
   ...:     raise Exception("This is my exception for Elad :)")
   ...: except Exception as ex:
   ...:     print(f"Exception raised: {ex.__str__()}")
   ...:
   ...:
Exception raised: This is my exception for Elad :)

In [3]: try:
   ...:     raise Exception("This is my exception for Elad :)")
   ...: except Exception as ex:
   ...:     print(f"Exception raised: {str(ex)}")
   ...:
   ...:
   ...:
Exception raised: This is my exception for Elad :)
```

Comment (ebenahar) [reply ocs_ci/ocs/utils.py]: @petr-balogh wasn't sure about this. Thanks for checking

**Comment by petr-balogh** [comment]
@vasukulkarni nope as when I ran today the verification it failed with another issue :  Error is Error from server (NotFound): pods "rook-ceph-mon-a-canary-76698dd7b7-4kvgn" not found
I am fixing this issue in different PR #669 .

**Comment by zmc** [comment]
The test build I started earlier has finished; no must-gather invocation took longer than 72s, so timeouts didn't even happen.

This is pretty confusing to me, considering neither ocs-must-gather or openshift's must-gather have changed since yesterday.

**Pull request merged**